app/src/main/java/com/teamagam/gimelgimel/app/mainActivity/view/MainActivityPanel.java

```diff
@@ -103,27 +109,57 @@ public void onPanelStateChanged(View panel, SlidingUpPanelLayout.PanelState prev
             sLogger.userInteraction("MainActivity's bottom panel mode changed from "
                     + previousState + " to " + newState);
             mViewModel.onChangePanelState(newState);
-        }
 
+            if (newState == SlidingUpPanelLayout.PanelState.ANCHORED) {
+                onPanelAnchored();
+            } else if (newState == SlidingUpPanelLayout.PanelState.COLLAPSED) {
+                onPanelCollapsed();
+            }
+        }
 
-        private void updateBottomPanelDimensions(float slideOffset) {
-            int height = calculateHeight(slideOffset);
-            adjustHeight(height);
+        private void updateBottomPanelDimensions(float bottomPanelSlideOffset) {
+            adjustBottomPanelContentHeight(getScreenWithoutCollapsedPanelHeight(),
+                    bottomPanelSlideOffset);
         }
 
-        private int calculateHeight(final float slideOffset) {
+        private int getScreenWithoutCollapsedPanelHeight() {
```

Comment:
I meant what it says, it's just that this isn't a constant because of screen orientation changes :S